src/tigerbeetle/cli.zig

```diff
@@ -21,7 +21,7 @@ const usage = fmt.comptimePrint(
     \\
     \\  tigerbeetle format --cluster=<integer> --replica=<index> <path>
     \\
-    \\  tigerbeetle start --addresses=<addresses> <path>
+    \\  tigerbeetle start --addresses=<addresses> --standby-count=<integer> <path>
```

Comment:
I now recall that jorangreef mentioned that we actually deliberately _don't_ have any optional arguments, and require all defaults to be specified explicitly (avoids surprises, and also doesn't _really_ reduce experience, as tigerbeetle is run by some automation, rather than manually).

Something to revisit when we converge on the final way to specify standbys and such.

Reply:
Actually, it'll be gone much earlier than that -- when I add replica_cont to format.